.github/workflows/test-e2e.yml

```diff
@@ -28,3 +28,18 @@ jobs:
           echo "${KUBECONFIG_FILE}" > $HOME/.kube/config
       - name: Run e2e tests
         run: make test-e2e
+
+      - name: Notify slack on failure
+        if: failure()
+        uses: ravsamhq/notify-slack-action@v2
+        with:
+          status: ${{ job.status }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          notification_title: "E2E test failure"
+          message_format: "{emoji} *{workflow}* {status_message} in <{run_url}|{repo}>"
+          # This is the core-app user group
+          mention_groups: "C077R275SRH"
+        env:
+          # This is an org level secret and is currently set to #releases in
+          # slack
```

Comment:
This comment looks incorrect because the Slack notifications don't go to a Slack channel called `#releases`.

```suggestion
```